wfe2/wfe.go

```diff
@@ -1416,8 +1416,10 @@ func (wfe *WebFrontEndImpl) Account(
 		return
 	}
 
-	// If the body was not empty, then this is an account update request.
-	if string(body) != "" {
+	// If the body was not either completely empty or an empty JSON object, then
+	// this is an account update request. Treating the empty JSON object like a
+	// POST-as-GET is a holdover from ACMEv1.
+	if string(body) != "" && string(body) != "{}" {
```

Comment:
Slight quibble: it's still reasonable in ACMEv2 to treat `"{}"` as an update to zero fields, returning the object. It's true that a more recently written client would be more likely to just use `""` if it knows its just trying to fetch the account. Perhaps this?

```suggestion
	// An empty string means POST-as-GET (i.e. no update). A body of "{}" means
	// an update of zero fields, returning the unchanged object. This was the
	// recommended way to fetch the account object in ACMEv1.
```